prisma/migrations/20230118160544_add_question_is_public/migration.sql

```diff
@@ -1 +0,0 @@
-ALTER TABLE `Questions` ADD COLUMN `is_public` BOOLEAN NOT NULL DEFAULT true;
```

Comment:
I think we will need to keep this migration and create a new one that drops the column. Otherwise there will be a drift in migration history. Another way would be keeping the column (but unused in app)